setup.py

```diff
@@ -20,7 +20,7 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.930",
+    "mypy>=0.930,<0.970",  # TODO: remove after CI is fixed
```

Comment:
This shouldn’t be necessary for the sake of CI. CI does `pip install -r ./requirements.txt`, and `requirements.txt` already includes `mypy==0.961` (and also `-e .[compatible-mypy]` which constrains `mypy` to `<0.970`).

Reply:
Any ideas how to fix it?